app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @add_new_comment = Comment.new
```

Comment:
- [ ] [optional] I would suggest changing of name to this variable and also consider using it in the create method , kindly consider applying the same in your posts controller new method
```suggestion
    `@comment = Comment.new`
    //and in the create method
    ``@comment = @current_user.comments.new(params)`
```